apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-settings/vo-settings-sponsored-members/vo-settings-sponsored-members.component.ts

```diff
@@ -0,0 +1,117 @@
+import { Component, OnInit } from '@angular/core';
+import { MembersManagerService, MemberWithSponsors, Vo } from '@perun-web-apps/perun/openapi';
+import { ActivatedRoute } from '@angular/router';
+import { SelectionModel } from '@angular/cdk/collections';
+import { FormControl } from '@angular/forms';
+import { TABLE_VO_MEMBERS } from '@perun-web-apps/config/table-config';
+import { MatDialog } from '@angular/material/dialog';
+import { getDefaultDialogConfig } from '@perun-web-apps/perun/utils';
+import { CreateSponsoredMemberDialogComponent } from '../../../../../shared/components/dialogs/create-sponsored-member-dialog/create-sponsored-member-dialog.component';
+import { GenerateSponsoredMembersDialogComponent } from '../../../../../shared/components/dialogs/generate-sponsored-members-dialog/generate-sponsored-members-dialog.component';
+import { Urns } from '@perun-web-apps/perun/urns';
+import { GuiAuthResolver } from '@perun-web-apps/perun/services';
+
+@Component({
+  selector: 'app-vo-settings-sponsored-members',
+  templateUrl: './vo-settings-sponsored-members.component.html',
+  styleUrls: ['./vo-settings-sponsored-members.component.scss']
+})
+export class VoSettingsSponsoredMembersComponent implements OnInit {
+
+  constructor(private membersManager: MembersManagerService,
+              private route: ActivatedRoute,
+              private dialog: MatDialog,
+              private authResolver: GuiAuthResolver) {
+  }
+
+  voId: number;
+  vo: Vo;
+  members: MemberWithSponsors[] = [];
+
+  createAuth: boolean;
+  generateAuth: boolean;
+
+  private attrNames = [
+    Urns.USER_DEF_ORGANIZATION,
+    Urns.USER_DEF_PREFERRED_MAIL,
+    Urns.MEMBER_DEF_ORGANIZATION,
+    Urns.MEMBER_DEF_MAIL,
+    Urns.MEMBER_DEF_EXPIRATION
+  ];
+
+  selection = new SelectionModel<MemberWithSponsors>(true, []);
+  searchString = '';
+  loading = false;
+  statuses = new FormControl();
+  statusList = ['VALID', 'INVALID', 'EXPIRED', 'DISABLED'];
+  selectedStatuses: string[] = ['VALID', 'INVALID', 'EXPIRED', 'DISABLED'];
```

Comment:
I dont understand why it is here

Reply:
This was used when this page was using member list, now it was useless so I deleted it.